features/templates.feature

```diff
@@ -12,3 +47,36 @@ Feature: Templates
           | template | number |
           |  valid   |   no   |
           |  invalid |  some  |
+
+    @ignore
+    Scenario Outline: Merges template into user's config
+        Given an existing pipeline with job main
+        And an existing template called Test
+        And user only specifies template Test in job main
```

Comment:
You may want this to match https://github.com/screwdriver-cd/screwdriver/pull/496/files#diff-2d113dc19bf2d9aa7571e6c04e00dfeaR63 for reusability